From 14604cd802ba36c7945fd0808f59cb509d0e8c14 Mon Sep 17 00:00:00 2001
Message-ID: <14604cd802ba36c7945fd0808f59cb509d0e8c14.1700222319.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1700222319.git.sharlatanus@gmail.com>
References: <cover.1700222319.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 15 Nov 2023 13:33:42 +0000
Subject: [PATCH 18/32] gnu: python-drms: Simplify package.

* gnu/packages/astronomy.scm (python-drms): Simplify package.
[build-system]: Swap to pyproject-build-system.
[arguments]: Drop them.

Change-Id: I9d8e82e63b850b220489454ba984f22f4cad3ad3
---
 gnu/packages/astronomy.scm | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index fa3ad52350..523030a0f7 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -1924,18 +1924,7 @@ (define-public python-drms
        (uri (pypi-uri "drms" version))
        (sha256
         (base32 "0mkrmr55fgca441z7hvsyri6x9cjsh0sfas3hrj0k1k10k8vszbw"))))
-    (build-system python-build-system)
-    (arguments
-     (list
-      #:phases
-      #~(modify-phases %standard-phases
-          (replace 'check
-            (lambda* (#:key inputs outputs tests?
-                      #:allow-other-keys)
-              (when tests?
-                (add-installed-pythonpath inputs outputs)
-                (setenv "JSOC_EMAIL" "jsoc@sunpy.org")
-                (invoke "python" "-m" "pytest" "-vv")))))))
+    (build-system pyproject-build-system)
     (native-inputs
      (list python-astropy
            python-pytest-astropy
-- 
2.41.0

